README.md

```diff
@@ -39,116 +29,137 @@
 
-### Run tests
+## 👥 Authors <a name="authors"></a>
+**Erwin Amador**
+- GitHub: [@ErwinAmador](https://github.com/eamador1)
+- Twitter: [@ErwinAmador](https://twitter.com/hokieamador)
 
-Use your browser's developer tools
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- AUTHORS -->
+## 🔭 Future Features <a name="future-features"></a>
 
-## 👥 Authors <a name="authors"></a>
+- [ ] **Use leaderboard API**
+- [ ] **Add Functionality**
+- [ ] **Add Styling**
 
-👤 **Esteban Palacios**
-
-- GitHub: [@Estete9](https://github.com/Estete9)
-- Twitter: [@NaughTban](https://twitter.com/NaughTban)
-- LinkedIn: [Esteban Palacios](https://www.linkedin.com/in/esteban-palacios-5030a772/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FUTURE FEATURES -->
+## 🤝 Contributing <a name="contributing"></a>
 
-## 🔭 Future Features <a name="future-features"></a>
+Contributions, issues, and feature requests are welcome!
 
-- **Gallery popup**
+Feel free to check the [issues page](../../issues/).
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## 👤 Collaboration:
 
-<!-- CONTRIBUTING -->
+Thank you Esteban for your collaboration in this project, especially your assistance with Webpack.
 
-## 🤝 Contributing <a name="contributing"></a>
+- GitHub: [@Estete9](https://github.com/Estete9)
 
-Contributions, issues, and feature requests are welcome!
+Thank you Alex, my coding partner, for being there when I have a doubt.
 
-Feel free to check the [issues page UPDATE URL HERE](https://github.com/Estete9/YOUR-REPO-NAME-HERE/issues).
+- GitHub: [@Estete9](https://github.com/alex1779)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- SUPPORT -->
-
 ## ⭐️ Show your support <a name="support"></a>
 
+If you like this project you can follow me on Github.
 
-If you like this project remember to star and share it 🥳️
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank all Microverse staff.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- ACKNOWLEDGEMENTS -->
+## ❓ FAQ <a name="faq"></a>
 
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
+- **What is a Linter?**
 
+  - Linter is a tool that analyzes source code to flag programming errors, bugs, stylistic errors, and suspicious constructs.
 
-- ACKNOWLEDGMENTS TO CREATIVE LICENSE AND OTHERS
+- **What is meant by Gitflow?**
+
+  - Gitflow is an alternative Git branching model that involves the use of feature branches and multiple primary branches.
+
+- **What is DOM?**
+
+  - The Document Object Model (DOM) is the data representation of the objects that comprise the structure and content of a document on the web. 
+  
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./MIT.md) licensed.
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Great job working on this milestone and updating the ReadMe. Unfortunately, when I click on the MIT license link, it gives me an error 404 page. It is not professional to allow broken links in our codebase. Kindly fix this issue.